rafthttp/transport.go

```diff
@@ -94,8 +94,10 @@ type Transporter interface {
 // User needs to call Start before calling other functions, and call
 // Stop when the Transport is no longer used.
 type Transport struct {
-	DialTimeout time.Duration     // maximum duration before timing out dial of the request
-	TLSInfo     transport.TLSInfo // TLS information used when creating connection
+	DialTimeout      time.Duration // maximum duration before timing out dial of the request
+	DialRetryTimeout time.Duration // alters the frequency of streamReader dial retrial attempts
```

Comment:
@xiang90 do you mean to have a single instance of ```DialRetryLimiter *rate.Limiter``` in ```rafthttp.Transport```? In this way ```DialRetryLimiter``` will be shared among all ```streamReader``` instances, so it will limit the dial frequency for every peer.

If this is a desired behavior, we definitely can put ```DialRetryLimiter``` into the ```Trasnport```, because ```rate.Limiter``` seems to be goroutine-safe.